### ISSUE TITLE
Versioning constraints wxpython2.8 and 'matplotlib>1.2.0,<1.5.0'

### ISSUE THREAD

**Issue by tillea** [mad-lab/transit#1]
Hi,
I'm a member of the Debian Med team which is a group inside Debian with the objective to package free software in life sciences for official Debian.  I intend to package transit which was requested by a user and I succeeded in packaging it and starting the program.  (Hint: The name tbb is choosen by some other program thus I renamed to transit-tbb - I'd recommend you should do this as well.)
Since you are explicitly specifying wxpython 2.8.0 I wonder whether wxpython 3.0 (which is available in recent Debian versions) will work flawlessly.  I also noticed that 'matplotlib>1.2.0,<1.5.0' is specified inside setup.py (not in the docs).  Could you please clarify the installation instructions about this?
Thanks, Andreas.

**Comment by mad-lab** [comment]
Hi Andreas,

We definitely had issues with previous versions of matplotlib < 2.0, which is why that range was specified in the setup.py. If that constraint wasn't mentioned in the documentation, then it's an oversight.

That said, I just tested TRANSIT using both wxpython 3.0 and matplotlib 2.0.0, on an ubuntu 16.04 machine, and TRANSIT appeared to work fine. It looks like the issues we had with matplotlib may have been fixed in the latest version.
